egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -384,17 +403,24 @@ def compute_preconditioning_matrix(dir, egs_dir, num_lda_jobs, run_opts,
     if max_lda_jobs is not None:
         if num_lda_jobs > max_lda_jobs:
             num_lda_jobs = max_lda_jobs
+    egs_string = ""
+    if select_feature_offset > -1:
+        egs_string = ("ark:nnet3-chain-copy-egs --select-feature-offset={0} "
+                      "ark:{1}/cegs.JOB.ark ark:- |".format(
+                      select_feature_offset,egs_dir))
```

Comment:
According to Google standards, this line should be indented 4 spaces ahead of where "egs_string =" began. It should not be in the same level as the string above.
Also add space after comma.